_data/kit_versions.yml

```diff
@@ -4,3 +4,5 @@
   released: 2021-11-20
 - version: 2022.0.0
   released: 2021-11-13
+- version: 2022.1.0
```

Comment:
New versions should be added to the top of the file, not the bottom (perhaps we should change that)

Reply:
Fixed